sdk/keyvault/azure-security-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultCertificatesTest.java

```diff
@@ -51,9 +51,10 @@ public void testGetCertificate() {
     }
 
     @Test
-    public void testRefreshAndGetAliasByCertificate() {
+    public void testRefreshAndGetAliasByCertificate() throws InterruptedException {
         Assertions.assertEquals(keyVaultCertificates.refreshAndGetAliasByCertificate(certificate), "myalias");
         when(keyVaultClient.getAliases()).thenReturn(null);
+        Thread.sleep(1);
```

Comment:
We can not let customer to use `Thread.sleep(1);`.
How about do like this:
```
    public static void updateLastForceRefreshTime() {
        lastForceRefreshTime = new Date(System.currentTimeMillis() + 1);
    }
```

Reply:
maybe we need remove other sleep code in test like [here](https://github.com/Azure/azure-sdk-for-java/blob/34a7365af5a1293dfb9ee2285e56df44345d025d/sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultKeyStoreTest.java#L171) and [here](https://github.com/Azure/azure-sdk-for-java/blob/34a7365af5a1293dfb9ee2285e56df44345d025d/sdk/keyvault/azure-security-test-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/KeyVaultCertificatesTest.java#L108) if we change this function

Reply:
Yes, please do that.